app/views/providers/has_national_insurance_numbers/show.html.erb

```diff
@@ -8,11 +8,8 @@
   <%= page_template page_title: t(".page_title"), template: :basic, form: do %>
 
     <%= form.govuk_radio_buttons_fieldset(:has_national_insurance_number,
-                                          legend: { text: page_title, size: "xl", tag: "h1" }) do %>
-
-      <p class="govuk-body govuk-!-margin-top-3 govuk-!-margin-bottom-6">
-        <%= t(".hint") %>
-      </p>
+                                          legend: { text: page_title, size: "xl", tag: "h1" },
+                                          hint: { text: Setting.special_childrens_act? ? nil : t(".hint") }) do %>
```

Comment:
Is this correct? Is the logic of whether or not to display the hint determined by the setting or whether the legal aid application has special children act proceedings? (ie something like legal_aid_application.special_children_act_proceedings?)

Reply:
I checked with Quay and yes, it's dependent on the sca feature flag. It'll be removed for all cases when the flag is enabled

Reply:
Ah OK, 👍